metricbeat/docs/modules/elasticsearch.asciidoc

```diff
@@ -7,6 +7,7 @@ This file is generated! See scripts/docs_collector.py
 
 This is the elasticsearch Module. The elasticsearch module contains a minimal set of metrics to enable monitoring of elasticsearch across multiple versions.
 
+For more detailed monitoring of elasticsearch use the https://www.elastic.co/guide/en/x-pack/current/xpack-monitoring.html[free x-pack monitoring].
```

Comment:
Can you create a variable in the index.asciidoc file and use the variable to resolve the path?

Add this to index.asciidoc:
`:monitoringdoc: https://www.elastic.co/guide/en/x-pack/current`

Then use `{monitoringdoc}/xpack-monitoring.html[free x-pack monitoring]`.

Note that for X-Pack we don't  use `{doc-branch}` to resolve the branch because we end up with broken links in master (for X-Pack, we don't build/publish docs in master).